src/main/java/org/dependencytrack/model/ConfigPropertyConstants.java

```diff
@@ -69,7 +70,18 @@ public enum ConfigPropertyConstants {
     KENNA_CONNECTOR_ID("integrations", "kenna.connector.id", null, PropertyType.STRING, "The Kenna Security connector identifier to upload to"),
     ACCESS_MANAGEMENT_ACL_ENABLED("access-management", "acl.enabled", "false", PropertyType.BOOLEAN, "Flag to enable/disable access control to projects in the portfolio"),
     NOTIFICATION_TEMPLATE_BASE_DIR("notification", "template.baseDir", SystemUtils.getEnvironmentVariable("DEFAULT_TEMPLATES_OVERRIDE_BASE_DIRECTORY", System.getProperty("user.home")), PropertyType.STRING, "The base directory to use when searching for notification templates"),
-    NOTIFICATION_TEMPLATE_DEFAULT_OVERRIDE_ENABLED("notification", "template.default.override.enabled", SystemUtils.getEnvironmentVariable("DEFAULT_TEMPLATES_OVERRIDE_ENABLED", "false"), PropertyType.BOOLEAN, "Flag to enable/disable override of default notification templates");
+    NOTIFICATION_TEMPLATE_DEFAULT_OVERRIDE_ENABLED("notification", "template.default.override.enabled", SystemUtils.getEnvironmentVariable("DEFAULT_TEMPLATES_OVERRIDE_ENABLED", "false"), PropertyType.BOOLEAN, "Flag to enable/disable override of default notification templates"),
+    TASK_SCHEDULER_LDAP_SYNC_PERIOD("task-scheduler", "ldap.sync.period","21600000", PropertyType.NUMBER, "Sync period (in milliseconds) for LDAP"),
```

Comment:
Nitpick: These properties should use "cadence" (as used for previously configurable tasks) or more ideally "interval", because they describe reoccurring events. A period describes a block of time, and interval the time between periods.

Reply:
Thanks @nscuro. I agree that cadence is more accurate.
I'll wait feedbacks from Steve before implementing the changes.

Reply:
All of these configurable task scheduler properties should not be in milliseconds. At the lowest, I would think minutes would be appropriate, however, I also think hour may work as well.

Reply:
I also think we need absolute minimum allowed. For example, I would not want GitHub mirroring to take place every minute. If this is going to be user configurable, we need to provide guardrails that allows flexibility in configuration without the ability to cause the system harm - which would increase support tickets.

Reply:
If we go that way, I think that hours would be more appropriate. The properties being positive numbers, it's fairly easy to set a minimum of one hour without having to configure it (property > 0).
I suppose that should be ok for now.
Should more control be given on the minimum allowed, it should be configurable through alpine, DTrack system property or DTrack environment variable according to me.